python/tests/integration/completions/chat_completion_test_base.py

```diff
@@ -58,39 +59,46 @@
 
 mistral_ai_setup: bool = False
 try:
-    MistralAIChatCompletion()
-    mistral_ai_setup = True
-except ServiceInitializationError:
+    if os.environ["MISTRALAI_API_KEY"]:
+        mistral_ai_setup = True
+except KeyError:
```

Comment:
I don't have a strong opinion on this but feels like this can be simplified:
```
def is_service_setup_for_testing(env_var_name: str) -> bool:
    """Check if 
    return env_var_name in os.environ and os.environ[env_var_name]

mistral_ai_setup = is_service_setup_for_testing("MISTRALAI_API_KEY")
...
```